README.md

```diff
@@ -17,6 +17,18 @@ variables can be customised.
 -   wagtail `>=2.15` if using `WagtailBaseForm`
 -   sass `>=1.33.0` if building the sass yourself
```

Comment:
At least some of these are out of date. (I'd never trust docs about this kind of thing anyway.)

Reply:
To my knowledge, they're all correct - which are wrong?

Reply:
Django and crispy-forms seems to be wrong, if I'm looking at the right place: https://github.com/torchbox/tbxforms/blob/4a9fd445008eac636f22c7b18681c8af3974f522/pyproject.toml#L60-L61

Reply:
🤦 thanks!

Reply:
Updated in https://github.com/torchbox/tbxforms/pull/80/commits/4c775273f39a90407822090cbb9c1c737d919cf0